pages/account/statistics.js

```diff
@@ -51,6 +57,11 @@ export async function getServerSideProps(context) {
     };
   }
 
+  //Linking a profile to an account is a one-time process. Subsequent logins will disregard any attempt to add the profile again.  
+  if (account) {
```

Comment:
I think this condition will always be true, because the login would fail

Reply:
You are saying the GitHub login will never fail right? I have added this just for the case where we will have our own signup and login page. But can be ignored for now.

Reply:
For now, let's remove if condition and call the method directly. Maybe in the future, we will see.

Reply:
> You are saying the GitHub login will never fail right

I mean if it fails, it would not get to this page, `next-auth` would redirect somewhere else before reach this code

Reply:
Ahaa. That I will take care in another issue https://github.com/EddieHubCommunity/LinkFree/issues/7349. Where the actual call be made. 

![image](https://github.com/EddieHubCommunity/LinkFree/assets/11041028/5ca3bea7-b864-4d13-9cfb-97dca7749c73)


See the highlighted code.

Reply:
Closed this comment in PR https://github.com/EddieHubCommunity/LinkFree/pull/8232